README.md

```diff
@@ -171,7 +171,9 @@ NOTES:
 |HLS       |**Y**     |**Y**    |**Y** ¹  | -        |
 
 NOTES:
- - ¹: We support FairPlay through Apple's native HLS player.
+ - ¹: We support FairPlay through MSE/EME when available and when
```

Comment:
Technically speaking, setting `useNativeHlsOnSafari` to true does not make us stop supporting FairPlay through MSE/EME. Whether we "support" something is a matter of if we CAN do it, rather than if we WILL do it.
I'd instead say:
```
By default, FairPlay is handled using Apple's native HLS player, when on Safari.
We do support FairPlay through MSE/EME, however.
See the `streaming.useNativeHlsOnSafari` configuration value.
```